src/components/SearchResults.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import TableInfo from "./TableInfo.js";
+import TableData from "./TableData.js";
+import FakeBookings from "./data/fakeBookings.json";
+
+function SearchResults() {
+  return (
+    <div>
+      <table className="table table-striped">
+        <TableInfo />
+        <TableData results={FakeBookings} />
```

Comment:
This is a nice split into smaller components👍

You're passing props to this component in `Bookings.js` but not using it here and importing data from json file again. You should keep one or the other - pass data as prop to `SearchResults` and then pass it down to `TableData` or remove the prop and import from `Bookings.js` and keep this file as is.